easy_profiler_core/reader.cpp

```diff
@@ -905,9 +909,9 @@ extern "C" {
                 //});
 
                 //root.tree.shrink_to_fit();
-                for (auto i : root.children)
+                for (auto child_block_index : root.children)
```

Comment:
How could I miss that :fearful:

Reply:
It was just a warning and fortunately i is not used inside the for loop, so i is merely hidden :)